components/QuestionController.tsx

```diff
@@ -38,6 +38,31 @@ const QUESTIONS = {
     ],
     title: 'Place your bet:',
   },
+  4: {
+    htmlFor: 'game1',
```

Comment:
would this still be `game1`?

Reply:
just copied how your wrote the others

Reply:
seems like the only thing that matters is the `key` which is `4` in this example

Reply:
that was just a refactor of how you made it. this was the previous version before I put it in a component. 
```
const questionThree = (
      <>
        <br></br>
        <label for='game1'>Place your bet:</label>
        <br></br>
        <select required onChange={this.onChange} value={answer} name='answer'>
          <option value=''>--Please choose an artist--</option>
          <option value='Fivio Foreign'>Fivio Foreign</option>
          <option value='Pop Smoke'>Pop Smoke</option>
        </select>
      </>
    );
```

Reply:
that `for` label can go foreal